### ISSUE TITLE
Is there a way to delete all markers from one of the crafted atlases?

### ISSUE THREAD

**Issue by vorwinator** [Hunternif/AntiqueAtlas#468]
I assume that there is no way to do it via GUI except creating new atlas 
but that would require me to run over the world again 
but that would delete all markers not only the death markers.
So is there some kind of file that I can edit to remove all markers of certain type 
or eventually a file that i can delete to remove all markers but keep the map?

**Comment by vorwinator** [comment]
world\data\aaMarkers_0.dat
So i found that this file can be deleted to remove all markers without removing map for number respective atlas 
but is it safe to do it?

**Comment by vorwinator** [comment]
![obraz](https://github.com/AntiqueAtlasTeam/AntiqueAtlas/assets/80983036/bc220718-9f30-428b-aef7-0c4dda10d12e)
![obraz](https://github.com/AntiqueAtlasTeam/AntiqueAtlas/assets/80983036/37b10713-2997-48da-8703-a8f56fa692d6)

**Comment by vorwinator** [comment]
Don't know if that's a good idea but it seems to be working

**Comment by tyra314** [comment]
I'd like to split this a bit into different concerns.

- First of all, why do you want to delete all markers? Or do you want to delete some markers, but it seems easier to delete all of them? Or are there markers you want to get rid of, but can't?
- To answer the question at hand, there's no in-game button to delete all markers, but you can delete individual markers by selecting the eraser button in the Atlas GUI and clicking on the marker you want to delete - either on the map or on the list of bookmarks on the left side.
- Yeah, editing or deleting the `aaMarkersXXX.dat` files should work. The `XXX` part is the number of the Atlas that you want to work on. Also look into the `DIM1`  and `DIM-1` folders, if you want to delete markers in the end or nether, respectively.

**Comment by vorwinator** [comment]
So this is my situation:
I'm playing RLCraft,
sharing atlas map with other players,
everyone has auto death marker ON,
one day there is a problem: im putting new marker on map and then it dissapears instantly after I clicked on "Done",
so I checked server logs and there was something about 1024 markers limit,
therefore I've changed this limit and started looking for ways to get rid of all death markers from the map.

1. I wanted to delete all death markers cause instead of map i had flood of death markers (would be awesome to have instead marker for last death location)

2. I would rather spend hours looking for automation than 40min on constant clicking with eraser. 
I'm not sure what bookmarks on the left are You talking about... perhaps it's something that I dont have in my current version (ss attached)
![obraz](https://github.com/AntiqueAtlasTeam/AntiqueAtlas/assets/80983036/33038c63-19ae-474b-9acf-22d71346c3f5)

3. Thanks for answer and tips! :D
